.travis.yml

```diff
@@ -26,7 +27,8 @@ matrix:
   fast_finish: true
 
 install:
-  - pip install --upgrade pip setuptools wheel tox tox-travis
+  - python -m ensurepip
```

Comment:
The Travis Python build environment surely provides pip so this seems redundant.

Reply:
I added ensurepip in n attempt to fix https://travis-ci.org/opencobra/cobrapy/jobs/649362347?utm_medium=notification&utm_source=github_status. It apparently didn't fix the problem. I'll remove it again.